src/extensions/rotation.rs

```diff
@@ -79,31 +90,32 @@ impl CustomConst for Constant {
     }
 
     fn const_type(&self) -> ClassicType {
-        match self {
+        let t: Type = match self {
             Constant::Angle(_) => Type::Angle,
             Constant::Quaternion(_) => Type::Quaternion,
-        }
-        .classic_type()
+        };
+        let t2: CustomType = t.into();
```

Comment:
This is a bit of a pain. Unfortunately a single `.into()` doesn't do a conversion requiring two calls of `.from` :(.

Reply:
But it can be at least a bit easier by using `.custom_type()`